assets/agenda/components/AgendaListItemIcons.jsx

```diff
@@ -117,8 +150,13 @@ AgendaListItemIcons.propTypes = {
     row: PropTypes.bool,
     isMobilePhone: PropTypes.bool,
     user: PropTypes.string,
+    listConfig : PropTypes.object,
 };
 
+const mapStateToProps = (state) => ({
+    listConfig: listConfigSelector(state),
+});
+
 AgendaListItemIcons.defaultProps = {isMobilePhone: false};
 
-export default AgendaListItemIcons;
+export default connect(mapStateToProps)(AgendaListItemIcons);
```

Comment:
the list config would be better passed from some higher level component, something like `AgendaList`

Reply:
okay